src/gt4py/next/program_processors/codegens/gtfn/itir_to_gtfn_ir.py

```diff
@@ -53,7 +54,7 @@ def pytype_to_cpptype(t: str):
             "float": "double",
             "float32": "float",
             "float64": "double",
-            "int": "int",
+            "int": "long",
```

Comment:
Here was another int->int, maybe that's why I changed the other. I agree with you that after the frontend all types should have concrete size and we should use the same types from a central place including their conversation to C++.